### DISCUSSION TITLE
Self-hosting installation

### DISCUSSION THREAD

**Discussion by amitnovick** [excalidraw/excalidraw#7056] [Q&A]
Hello,

I wish to self-host the web app showcased at Excalidraw.com.

Could you please list instructions on how to do it?

# --

A few things I've seen from looking at the project files (feel free to ignore these observations if they're not relevant):

In `package.json` I see several scripts:
- "build-node": "node ./scripts/build-node.js"
- "build:app:docker": "cross-env VITE_APP_DISABLE_SENTRY=true VITE_APP_DISABLE_TRACKING=true vite build"
- "build:app": "cross-env VITE_APP_GIT_SHA=$VERCEL_GIT_COMMIT_SHA vite build"

and so on.

Also, I see a Dockerfile and docker-compose.yml file.

Do you provide a Docker image as well?

# -- 

Thanks,
Amit

**Comment by wjxiu** [comment]
docker official image 
https://hub.docker.com/r/excalidraw/excalidraw
